### ISSUE TITLE
Visual stimulus overdose

### ISSUE THREAD

**Issue by arnoldmashava** [bluz71/vim-nightfly-colors#55]
![Screenshot_20230502_023914](https://user-images.githubusercontent.com/5759101/235556821-3b18d423-490b-4565-bb6f-663359d2dce9.png)

```vimscript
let g:nightflyTransparent = v:true
```

My faculties are levitating in the Mystique...

**Comment by bluz71** [comment]
I presume you are pleased?
Or is there an issue?

Hard for me to read between this lines.

Cheers.

**Comment by arnoldmashava** [comment]
@bluz71 In simpler terms: herewith my heartfelt acknowledgements and highest regards, for your seemingly infinite creativity and the seamless ability to translate your visualizations into deployable `Vimscript` and `Lua` code, for the sensual betterment and aesthetic enjoyability of coding and debugging through an otherwise insipid Modal Code Editor. 

Of course, I am totally blown away by too much of a good thing - your visual perception and presentation of colors feels somewhat Spiritual: against the Midnight Blue background, the highlights from the Purple got me lost in the Haze and I keep coming back for more:

1. compatibility with both Vim and Neovim 

2. tweaking flexibility and comprehensive customization options that are easy to implement

```vimscript
let g:nightflyCursorColor = v:true
let g:nightflyTransparent = v:true
let g:nightflyUnderlineMatchParen = v:true
```

3. flawless integration with and support for the widely utilized Statuslines

```vimscript
let g:lightline = { 'colorscheme': 'nightfly' }
```

[Lualine](https://github.com/nvim-lualine/lualine.nvim)

[vim-mistfly-statusline](https://github.com/bluz71/vim-mistfly-statusline)
```vimscript
if g:colors_name == 'moonfly' || g:colors_name == 'nightfly'
        " Do nothing since both colorschemes already set mistfly mode colors.
        return
```
4. perfectly constructed and readable documentation

5. constantly maintained code repositories through fully satisfied `git commits`

In conclusion, unless if you intend to keep Issue #55 Open, I have no objections to your resolution of the "Visual stimulus overdose" note of appreciation as Closed. Of course, I fully concede and would be very happy if you can retain and use my pictorial exhibit for illustrative purposes.

Thank you so much for your immense contributions to our perpetually cryptic Universe of GNU/Linux and BSD.

Muchas Gracias y Saludos Cordiales...

**Comment by bluz71** [comment]
Thank you, that is quite an abounding testimonial indeed.

Take care and enjoy.

Best regards.
